### PR TITLE
update .gitignore for all

### PR DESCRIPTION
**Pull request by ToxEn**

### CODE REVIEW

File: app/config/parameters.yml.dist [hunk 1]
```diff
@@ -22,4 +22,4 @@ parameters:
     api_description:   The full description of your API
 
     # A secret key that's used to generate certain security-related tokens
-    secret:            ThisTokenIsNotSoSecretChangeIt
+    secret:            ThisTokenIsNotSoSecretChangeIt
```
Comment (Evergreenn) [inline app/config/parameters.yml.dist]: :3

File: .gitignore [hunk 2]
```diff
@@ -1,12 +1,47 @@
...
+/phpunit.xml
+
+# Build data
 /build/
-/vendor/
-/bin/
+
+# Composer PHAR
 /composer.phar
+
```
Comment (pimolo) [inline .gitignore]: IMO `.gitignore` file should be application-specific. So you should delete `.idea`and all this shit

File: app/config/config.yml [hunk 3]
```diff
@@ -44,7 +44,7 @@ doctrine:
         # if using pdo_sqlite as your database driver:
         #   1. add the path in parameters.yml
         #     e.g. database_path: "%kernel.root_dir%/data/data.db3"
-        #   2. Uncomment database_path in parameters.yml.dist
```
Comment (pimolo) [inline app/config/config.yml]: keep that

File: .gitignore [hunk 4]
```diff
@@ -1,12 +1,47 @@
...
+!bin/symfony_requirements
+/vendor/
+
+# Assets and user uploads
+/web/bundles/
+/web/uploads/
+
+# PHPUnit
 /app/phpunit.xml
+/phpunit.xml
```
Comment (pimolo) [inline .gitignore]: we don't use it

File: .gitignore [hunk 5]
```diff
@@ -1,12 +1,47 @@
...
+
+# Build data
 /build/
```
Comment (pimolo) [inline .gitignore]: same

File: .gitignore [hunk 6]
```diff
@@ -1,12 +1,47 @@
...
-/app/bootstrap.php.cache
+# Created by .ignore support plugin (hsz.mobi)
+### Example user template template
+### Example user template
+
+# IntelliJ project files
+.idea
+*.iml
+out
+gen### Symfony template
```
Comment (pimolo) [inline .gitignore]: new line

**Comment by pimolo** [comment]
Nope.

**Comment by Evergreenn** [comment]
:+1:

**Comment by Evergreenn** [comment]
Nope. @ToxEn  You need to integrate Travis to your commit

File: .travis.yml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+
+before_script:
+  - composer install --dev --prefer-source
+
+script: bin/behat
```
Comment (Evergreenn) [inline .travis.yml]: :3

**Comment by Evergreenn** [comment]
:+1:

**Pull request merged**